### DISCUSSION TITLE
SBS 180VR Image output?

### DISCUSSION THREAD

**Discussion by AVVJIX** [thygate/stable-diffusion-webui-depthmap-script#373] [Ideas]
Hello! Awesome script, it works like a champ!

I have been exporting lots of files but mostly Stereoscopic images. The side-by-side images work in my VR viewer (GizmoVR) but the image isn't "immersive." I tied out a tool called 3DCombine you can export SBS 180vr images which gives it that immersive feel (although it's small). I am wondering if the 180VR image map could be an export option (that distorted half-circle type view).

Just in case anyone wants a quick guide on 3Dcombine, here are the steps:
-Click file, open images, 3D
-Pick your SBS image from this script
-Choose Parallel, click OK
-Click the VR on the top bar, then Oculus on the sidebar
-.40 is the default size, .6 is a little more zoomed in (this is what I pick), .8 or 1 then the image is Distored in the viewer, click OK
-180 Oculus Quest Gallary, click OK
-File, Save 3D, then click the Same image button on the middle bar, name the file whatever you like

Transfer the image to your phone, save the file somewhere in the main hard drive (not the SD card, at least for the viewer I use). Open cardboard, open GizmoVR (or install that), find your image, click the gear icon on the bottom right, change the settings to the side-by-side icon, and 180

Hey hey and you have semi-immersive 3d images!! You can also open the side-by-side images this script exports without using 3dcombine, and it works in the viewer as is, but it's a "flat" image.

**Comment by Vega-KH** [comment]
Nice tip, and sometimes it does look better with that fisheye effect.

This gave me an idea. I wonder if I can train an SDXL LoRA on vr180 fisheye images, then use SD to outpaint the black area of the image...
